stories/setup/styles.scss

```diff
@@ -0,0 +1,53 @@
+@import '~amo/css/styles';
+
+html,
+body {
+  // This overrides the App background style which makes it hard to see
+  // in storybook.
+  background: #fff;
+  // Giving extra cushion so nothing gets covered.
+  padding: 24px 12px 12px;
+}
+
+// Setting up chapter styles.
+.section-container {
+  border-bottom: 1px solid $background-grey;
+  display: block;
+  flex-wrap: wrap;
+  padding: 24px;
+
+  @include respond-to(medium) {
+    align-items: center;
+    display: flex;
+  }
+
+  .section-header {
+    flex-basis: 100%;
+    flex-grow: 1;
+    margin-bottom: 12px;
+  }
+
+  .section-component-container {
+    margin-top: 12px;
+  }
+
+  .section-subtitle {
+    color: #bbb;
+
+    @include respond-to(large) {
+      margin: 0;
+    }
+  }
+
+  .section-button-container {
+    float: left;
```

Comment:
I can't find `.section-button-container` in the rendered page but could this use flex instead? Floats are hard to get right.

Reply:
actually I think this is when I was showing the toggle source (which I've since removed). I had to override their style (despite the useTheme false setting we had). Anyway, it's moot at this point so I'll remove this : )